src/main/java/com/google/alpollo/model/SongSentiment.java

```diff
@@ -1,19 +1,24 @@
 package com.google.alpollo.model;
 
+import com.google.alpollo.AnalysisHelper;
+
 /**
- * Class created to store only sentiment score and magnitude, leaving the other useless properties
- * of the Sentiment object.
- * Class is mainly used for converting its instances to JSON strings.
+ * Class created to store only sentiment score and magnitude, leaving the other
+ * useless properties of the Sentiment object. Class is mainly used for
+ * converting its instances to JSON strings.
  */
 public class SongSentiment {
   /** Each sentiment has a score, showing the overall positivity of the text, ranging from -1.0 to 1.0. */
   private final float score;
   /** Each sentiment has a magnitude, representing how strong the sentiment is, ranging from 0.0 to +inf. */
   private final float magnitude;
+  /** Based on the score and magnitude of a sentiment, we can make up an interpretation of these values. */
+  private final String interpretation;
```

Comment:
I am assuming you'll need a getter for this at some point. Better to add it in this PR as well if that's the case